### DISCUSSION TITLE
lower difficulty multiplier based on mod customisation

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#20226] [Ideas] [answered]
The difficulty multiplier should get adjusted based on current mod customisation.
Why is it a problem if it doesn't get adjusted?
As of now (osu! 2022.902.1-lazer) the mods wind up and wind down can both be adjusted to go from 0.5-0.51x speed without any score penalty / lower difficulty modifier, effectively giving an option to play at the speed of halftime/daycore (0.74-0.75x speed in mod customisation) without the 0.5x difficulty modifier being applied, making it possible to get the full 1,000,000 score at speeds as low as 0.5x.
I am sure that there is similar exploits available with other mods that offer customisation as well, which is why the difficulty modifier should be adjusted based on mod customisation as well and not only based on mod selection.

**Comment by Flamiii** [comment, marked as answer]
Wind Up / Wind Down mod multipliers were addressed in #20206 (adjusted to 0.5x)
